beacon-chain/blockchain/process_block.go

```diff
@@ -318,9 +318,6 @@ func (s *Service) onBlockBatch(ctx context.Context, blks []block.SignedBeaconBlo
 		// Save potential boundary states.
 		if slots.IsEpochStart(preState.Slot()) {
 			boundaries[blockRoots[i]] = preState.Copy()
-			if err := s.handleEpochBoundary(ctx, preState); err != nil {
```

Comment:
`handleEpochBoundary` does more. It fills proposer and committee cache. Are you sure it's ok to remove it?

It's probably ok because those caches have fill-on-miss right?